src/main/java/gov/usgs/cida/pubs/busservice/sipp/SippProcess.java

```diff
@@ -111,9 +111,15 @@ protected InformationProduct getInformationProduct(String ipNumber) {
 				informationProduct.setPublicationType(idsPubTypeConv.getPublicationType());
 				informationProduct.setPublicationSubtype(idsPubTypeConv.getPublicationSubtype());
 
-				informationProduct.setUsgsSeriesTitle(
-						getSeriesTitle(idsPubTypeConv.getPublicationSubtype(),
+				if (StringUtils.isNotBlank(informationProduct.getJournalTitle())
+						&& PubsUtils.isPublicationTypeArticle(idsPubTypeConv.getPublicationType())
+						&& null != idsPubTypeConv.getPublicationSubtype()) {
```

Comment:
this looks similar to what I posted prior. I would move the not null check again.
If they are identical then I would make a helper function. They look identical to me.

Reply:
The results of each condition perform mostly the same code except for the inner most actions: getJournalTitle and getUsgsSeriesType.
I would recommend moving the common code out of the conditional and placing the results of the inner most action in a variable.
Turns out multiple line code suggestions are not clean or I would post what I mean.